std_msgs/Header header
std_msgs/Float64[] cartesian_collision
std_msgs/Float64[] cartesian_contact
std_msgs/Float64[] q
std_msgs/Float64[] dq
std_msgs/Float64[] tau_J
std_msgs/Float64[] dtau_J
std_msgs/Float64[] EE_F_ext_hat_EE
std_msgs/Float64[] elbow_start
std_msgs/Float64[] joint_collision
std_msgs/Float64[] joint_contact
std_msgs/Float64[] O_F_ext_hat_EE
std_msgs/Float64[] q_d
std_msgs/Float64[] q_start
std_msgs/Float64[] tau_ext_hat_filtered
std_msgs/Float64MultiArray O_T_EE_start
